source/documentation/administration_guide/topics/Host_Power_Management_settings_explained.adoc

```diff
@@ -10,7 +10,7 @@ The *Power Management* settings table contains the information required on the *
 |===
 |Field Name |Description
 |*Enable Power Management* |Enables power management on the host. Select this check box to enable the rest of the fields in the *Power Management* tab.
-|*Kdump integration* |Prevents the host from fencing while performing a kernel crash dump, so that the crash dump is not interrupted. In {enterprise-linux} 7.1 and later, kdump is available by default. If kdump is available on the host, but its configuration is not valid (the kdump service cannot be started), enabling *Kdump integration* will cause the host (re)installation to fail. If this is the case, see xref:fence_kdump_Advanced_Configuration[fence_kdump Advanced Configuration].
+|*Kdump integration* |Prevents the host from fencing while performing a kernel crash dump, so that the crash dump is not interrupted. In {enterprise-linux} 7.1 and later, kdump is available by default. If kdump is available on the host, but its configuration is not valid (the kdump service cannot be started), enabling *Kdump integration* will cause the host (re)installation to fail. If this is the case, see xref:fence_kdump_Advanced_Configuration[fence_kdump Advanced Configuration]. If you enable Kdump Integration or Disable Kdump Integration, you must perform Reinstall operation on the host in order to apply this option to the host.
```

Comment:
I just noticed that this admonition is already in [Setting fencing parameters](https://www.ovirt.org/documentation/administration_guide/index.html#sect-Host_Resilience). I recommend you copy it exactly. Tahlia Richardson wrote this line in 2018 and it is exactly how I would have phrased it.

![image](https://user-images.githubusercontent.com/31725355/169230248-4ab5f0d2-41cf-4bda-ac42-ad8585d47802.png)

Reply:
Also, I recommend removing "If this is the case, see xref:fence_kdump_Advanced_Configuration[fence_kdump Advanced Configuration]." It does not make sense to me. 

If you include the link in the admonition, I do not think you need the "If this is the case..."

`If you enable or disable *Kdump integration* on an existing host, you must xref:Reinstalling_Hosts_admin[reinstall the host].`

Reply:
+1

Reply:
Done